### PR TITLE
Unescape urls extract email v2

### PR DESCRIPTION
**Pull request by yucohen**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/28913

## Description
Emails like example[@]example.com are now extracted, and urls like meows://www.google.com are supported.

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CommonScripts/ReleaseNotes/1_4_27.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Scripts
+##### ExtractEmailV2
+- Email addresses with [@] are now extracted.
```
Comment (amshamah419) [inline Packs/CommonScripts/ReleaseNotes/1_4_27.md:4]:
```suggestion
- Email addresses containing [@] are now extracted.
```

File: Packs/CommonScripts/ReleaseNotes/1_4_27.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+##### UnEscapeURLs
+- Urls with meow:// and meows://.
```
Comment (amshamah419) [inline Packs/CommonScripts/ReleaseNotes/1_4_27.md:6]:
```suggestion
- URLs with `meow://` and `meows://` are now supported.
```

File: Packs/CommonTypes/ReleaseNotes/3_1_12.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+
+#### Indicator Types
+- **emailRep** - added to regex emails with [@] as valid .
```
Comment (amshamah419) [inline Packs/CommonTypes/ReleaseNotes/3_1_12.md:3]:
```suggestion
- **emailRep** - The regex now recognizes emails with `[@]` as valid.
```

**Comment by amshamah419** [comment]
@yucohen Just commit the suggested changes and you're good to go.

**Review by teizenman — APPROVED** [review]

**Pull request merged**